css/carouselImages.css

```diff
@@ -0,0 +1,191 @@
+  height: 3.75rem; /* 60px */
+  text-align: center;
+  border: 0.125rem solid var(--main-color); /* 2px */
+  background-color: var(--light-color);
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: opacity 0.15s ease, background-color 0.15s ease,
+    border-color 0.15s ease;
+}
+
+.carousel-control-prev-custom:hover,
+.carousel-control-prev-custom:focus,
+.carousel-control-next-custom:hover,
+.carousel-control-next-custom:focus {
+  background-color: var(--secondary-hover-color);
+}
+
+.carousel-control-prev-icon,
+.carousel-control-next-icon {
+  width: 0.9375rem; /* 15px */
+  height: 0.9375rem; /* 15px */
+  display: inline-block;
+  position: relative;
+}
+
+.carousel-control-prev-icon::before,
+.carousel-control-next-icon::before {
+  content: "";
+  width: 100%;
+  height: 100%;
+  border: solid var(--main-color);
+  border-width: 0 0.125rem 0.125rem 0; /* 2px */
+  display: inline-block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.carousel-control-prev-custom .carousel-control-prev-icon::before {
+  transform: translate(-50%, -50%) rotate(135deg);
+}
+
+.carousel-control-next-custom .carousel-control-next-icon::before {
+  transform: translate(-50%, -50%) rotate(-45deg);
+}
+
+/* ================== CAROUSEL IMAGE SLIDER ================================================================= */
+
+.flickity-page-dots {
+  display: none !important;
+}
+
+.gallery-cell {
+  width: 714px;
+  height: 31.25rem; /* 500px */
+  margin-right: 50px;
+  border-radius: 30px;
+  overflow: hidden;
+  transition: transform 0.2s ease;
+}
+
+.gallery-cell img {
+  width: 714px;
+  height: 545px;
+  object-fit: cover;
+  border-radius: 30px;
+}
+
+.flickity-viewport {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 60px;
+}
+
+/* ============== BUTTON CAROUSEL BOTTOM =================================================================== */
+
+.carousel-btn {
+  display: block;
+  margin: 75px auto 0 auto;
+  padding: 0.75rem 1.875rem !important; /* 12px 30px */
+  background-color: var(--primary-color);
+  border: 0.0625rem solid transparent; /* 1px */
+  border-radius: 2.5rem; /* 40px */
+  box-shadow: var(--shadow-color) 0 0.0625rem 0.1875rem 0; /* 0 1px 3px 0 */
+  box-sizing: border-box;
+  color: var(--light-color);
+  font-family: "Poppins", sans-serif;
+  font-size: 1rem; /* 16px */
+  line-height: 1.25;
+  min-height: 3rem;
+  position: relative;
+  transition: all 250ms;
+  -webkit-user-select: none;
+  touch-action: manipulation;
+  vertical-align: baseline;
+  width: auto;
+}
+.carousel-btn:hover,
+.carousel-btn:focus {
+  background-color: var(--primary-hover-color);
+  color: var(--button-hover-color);
+}
+
+.carousel-btn:active {
+  background-color: var(--primary-hover-color);
+}
+
+/* =============================== MEDIA QUERY =========================================================== */
+@media (max-width: 768px) {
```

Comment:
**Refactor Media Queries:**
   - Combine similar media query conditions to reduce redundancy and improve readability.

   ```css
   @media (max-width: 768px) {
     .carousel-section {
       padding-top: 4rem; /* 64px */
       padding-bottom: 4rem; /* 64px */
     }

     .carousel-title {
       font-size: 2rem; /* 32px */
     }

     .carousel-control-prev-custom,
     .carousel-control-next-custom {
       width: 3rem; /* 48px */
       height: 3rem; /* 48px */
     }

     .gallery-cell,
     .gallery-cell img {
       margin-right: 10px;
       margin-left: 10px;
       width: 100%;
       height: auto; /* Adjusted for responsiveness */
     }
   }
   ```